charts/mattermost-push-proxy/values.yaml

```diff
@@ -1,10 +1,10 @@
 replicaCount: 1
 image:
   repository: mattermost/mattermost-push-proxy
-  tag: 5.30.0
```

Comment:
could you elaborate why did we remove the tag?

Reply:
ignore, you used the app version.

Reply:
Because I added the default value from `appVersion` in the `Chart.yaml` . We do not have to change the version in both places

Reply:
```yaml
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```